# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/cookiestore/CookieStore.h
# Commit: 2081d42e6ccd
# Full Hash: 2081d42e6ccdcc39b3d1b2e6523a8d44fed85390
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-13 04:28:43
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 8 - CookieStore API - e10s support, r=smaug,cookie-reviewers,timhuang
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D217574
# ==============================================================================

diff -r c6e20b939a21 -r 2081d42e6ccd dom/cookiestore/CookieStore.h
--- a/dom/cookiestore/CookieStore.h	Thu Sep 12 16:48:21 2024 +0000
+++ b/dom/cookiestore/CookieStore.h	Thu Sep 12 16:48:21 2024 +0000
@@ -17,6 +17,7 @@
 
 class CookieData;
 class CookieStoreChild;
+class CookieStoreNotificationWatcherWrapper;
 class CookieStoreNotifier;
 class Promise;
 
@@ -72,6 +73,7 @@
 
   RefPtr<CookieStoreChild> mActor;
   RefPtr<CookieStoreNotifier> mNotifier;
+  RefPtr<CookieStoreNotificationWatcherWrapper> mNotificationWatcher;
 };
 
 }  // namespace mozilla::dom